### ISSUE TITLE
[@types/node] error TS2430: Interface 'NodeJS.Module' incorrectly extends interface '__WebpackModuleApi.Module'.

### ISSUE THREAD

**Issue by budarin** [DefinitelyTyped/DefinitelyTyped#47602]
- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for Gitter.
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @jkomyno @a-tarasyuk @alvis @r3nya @btoueg @brunoscheufler @smac89 @touffy @DeividasBakanas @eyqs @Flarna

```bash
node_modules/@types/node/ts3.1/globals.d.ts:573:15 - error TS2430: Interface 'NodeJS.Module' incorrectly extends interface '__WebpackModuleApi.Module'.
  Types of property 'parent' are incompatible.
    Type 'Module | null | undefined' is not assignable to type 'NodeModule | null'.
      Type 'undefined' is not assignable to type 'NodeModule | null'.

573     interface Module {
                  ~~~~~~
```

**Comment by euanmillar** [comment]
I am also getting this.  "parent" type is incompatible with webpack
![Screenshot 2020-09-15 at 20 02 04](https://user-images.githubusercontent.com/3169954/93253135-5dd6db00-f78e-11ea-843e-e99ab0400b48.png)

**Comment by Flarna** [comment]
I think https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47516 should fix this.

**Comment by jamescdavis** [comment]
Fixed in [@types/webpack-env@1.15.3](https://www.npmjs.com/package/@types/webpack-env/v/1.15.3)
